setup.cfg

```diff
@@ -1,5 +1,18 @@
+[metadata]
+name = Tubes
+version = 0.2.0
+description = Flow control and backpressure for event-driven applications.
+license = MIT
+url = https://github.com/twisted/tubes/
+
+[options]
+packages = find:
+install_requires =
+    six
+    Twisted
+
 [bdist_wheel]
-universal=1
+universal = 1
```

Comment:
FTR this produces a `py2.py3` tag. Does this project still support Python 2? If not, this should be `0` (or removed).

Reply:
It still does for now but it should probably be removed soon after landing this.